extras/upgrade_activities.sh

```diff
@@ -158,7 +178,10 @@ else
 fi
 
 # kill gluster-blockd
-pkill -15 gluster-blockd
+kill -15 ${GBD_PID} > /dev/null 2>&1
+
+# let's wait until the child process is finished
+wait ${GBD_PID}
```

Comment:
should we also do the same here ? if GBD_PID -gt 0; kill and wait ?
Or may be create a function like
```
function kill_N_wait()
{
  local pid=${1}

  kill -15 ${pid}
  wait ${pid}
}
```

Reply:
I think this is okay, since it is here we can be sure that the daemon is already running.
But it maybe always better to have this check.